doc/ccxt.pro.manual.rst

```diff
@@ -1456,7 +1456,7 @@ If your application is not very time-critical, you can still subscribe to OHLCV
    if (exchange.has['watchOHLCV']) {
        while (true) {
            try {
-               const candles = await exchange.watchOHLCV (symbol, since, limit, params)
+               const candles = await exchange.watchOHLCV (symbol, timeframe, since, limit, params)
```

Comment:
The .rst file is generated, please move your edits to the .md file.

Reply:
The .md file is already modified, but there's still no changes here: https://docs.ccxt.com/#/ccxt.pro.manual?id=watchohlcv

Reply:
@yzh-pelle have fixed the docs server, should be there now